app.js

```diff
@@ -490,6 +502,7 @@ function setRandomLifeStatus(status) {
  * @return {Array} - The modified generation containing a new live cell.
 */
 function addRandomLife(generation) {
+  
```

Comment:
I think more random squares would be cool; not sure if you're saving that for next iteration